gradle/scripts/build-flags.gradle

```diff
@@ -35,7 +35,7 @@ if (!projectsWithFlags('java').isEmpty()) {
     }
 
     if (!projectsWithFlags('java', 'relocate').isEmpty()) {
-        ['com.github.jengelman.gradle.plugins:shadow', 'com.guardsquare:proguard-gradle'].each {
+        ['gradle.plugin.com.github.johnrengelman:shadow', 'com.guardsquare:proguard-gradle'].each {
```

Comment:
These changes have been introduced in armeria but never updated back upstream 😅 

https://github.com/line/gradle-scripts/pull/147

Reply:
Oopsie! Thanks, @jrhee17